src/discussions/posts/post-filter-bar/PostFilterBar.jsx

```diff
@@ -29,9 +24,11 @@ function PostFilterBar({
 }) {
   const dispatch = useDispatch();
   const currentSorting = useSelector(selectThreadSorting());
-  const currentStatusFilter = useSelector(selectThreadStatusFilter());
-  const currentAllPostsTypeFilter = useSelector(selectThreadAllPostsTypeFilter());
-  const currentMyPostsTypeFilter = useSelector(selectThreadMyPostsTypeFilter());
+  const {
+    statusFilter: currentStatusFilter,
+    allTypeFilter: currentAllPostsTypeFilter,
+    myTypeFilter: currentMyPostsTypeFilter,
```

Comment:
I think if we give them meaningful name in reducer we wont be needing it to rename here

Reply:
I don't think that's true here. If you're dealing with two filter values, then you do need to disambiguate them somehow. Here I'm adding current to clarify that it's the current value. In most other places that implicit.